homeassistant/components/amcrest/services.yaml

```diff
@@ -73,3 +73,16 @@ stop_tour:
     entity_id:
       description: "Name(s) of the cameras, or 'all' for all cameras."
       example: 'camera.house_front'
+
+ptz_control:
+  description: Move (Pan/Tilt) and/or Zoom a PTZ camera
+  fields:
+    entity_id:
+      description: "Name of the camera, or 'all' for all cameras."
+      example: 'camera.house_front'
+    movement:
+      description: "up, down, right, left, top_right, bottom_right, top_left, bottom_left, zoom_in, zoom_out"
+      example: 'right'
+    travel_time:
+      description: "(optional) Incremental travel time in fractional seconds: from 0 to 1.  Default:  .2"
```

Comment:
Why "Incremental"? Not sure what you mean by that. Why not just "Travel time..."?

Reply:
agree it is not meaningful.    removed.